db/migrate/20230403093313_add_user_ref_to_orders.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToOrders < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :orders, :user, foreign_key: true
```

Comment:
`foreign_key` not required here

Reply:
Sir, by default foreign_key defaults to false